### ISSUE TITLE
[Tech Debt]: support accessibility testing in cypress

### ISSUE THREAD

**Issue by christianvogt** [opendatahub-io/odh-dashboard#2252]
### Type

Quality (eg. tests)

### Source

main

### Description

Accessibility testing is integrated into our playwright tests today. The move to cypress must maintain equivalent accessibility testing and report violations.

### Why?

Development must adhere to the accessibility web standards and scanning for violations against the built app is the best way to conform.

**Comment by adnankhan666** [comment]
https://issues.redhat.com/browse/RHOAIENG-1779
